lib/booker/v4.1/merchant.rb

```diff
@@ -127,6 +127,22 @@ def customer(id:, params: {})
         get("#{V41_PREFIX}/customer/#{id}", build_params(additional_params, params), Booker::V4::Models::Customer)
       end
 
+      def update_customer(id:, update_params: {})
+        additional_params = {
+            LoadUnpaidAppointments: false,
+            includeFieldValues: false
+        }
+        # get a raw json response because we need to send all fields back with modifications
+        customer_response = get("#{V41_PREFIX}/customer/#{id}", build_params(additional_params), nil)
```

Comment:
We should utilize the function wrappers around these api requests, you can update customer to accept an optional param on whether to return raw json response or not